UndertaleModTool/Windows/TextInput.cs

```diff
@@ -111,14 +111,24 @@ private void TextInput_Load(object sender, EventArgs e)
 
             if (richTextBox1.ReadOnly)
             {
-                richTextBox1.BackColor = TextBoxBGColor; //restore color to default one.
+                richTextBox1.BackColor = TextBoxBGColor; // restore color to default one.
                 richTextBox1.ContextMenuStrip = textCopyMenu;
             }
         }
 
         private void richTextBox1_TextChanged(object sender, EventArgs e)
         {
         }
+        private void richTextBox1_KeyDown(object sender, KeyEventArgs e)
+        {
+            // Prevent image pasting
+            if (e.Control && e.KeyCode == Keys.V)
```

Comment:
this seems bad. is there no way to check whether the user has input a "paste" combination? because this might not be necessarily CTRL+V if you change the combo in windows, or have a different combo in linux and run via WINE.

Reply:
Hmm, I didn't know that you can change that

Reply:
Wait, it seems you can't change that in Windows - https://answers.microsoft.com/en-us/windows/forum/all/need-to-change-the-shortcut-keys-for-copypaste/87583aa7-68c4-4b22-9cc0-d298b537a094.
Though, there is an alternative insert combination - `Shift+Insert`.

Also, are you sure that Wine doesn't emulate `Ctrl+V`?

Reply:
Anyway, I added the second combination, and I don't know a better way to detect the paste action there.

Reply:
> it seems you can't change that in Windows 

You can: 
![grafik](https://github.com/krzys-h/UndertaleModTool/assets/38186597/6d04b840-4c50-4dd7-81b5-cd6ac1f39d26)
https://learn.microsoft.com/en-us/windows/powertoys/keyboard-manager

> Also, are you sure that Wine doesn't emulate Ctrl+V?

WINE passes the paste signal through. And linux DE's let you customize anything you want as paste. from META+V to F1 to CTRL+SHIFT+INSERT+PAGEUP+F12+F1+Z+SPACE

Reply:
cant we somehow intercept the paste event, and delete image contents instead? Am not a fan of hardcoding key combinations.

Reply:
can you please unresolve this? @VladiStep

Reply:
I just checked, PowerToys sends `Ctrl+V` to the app in the end anyway, so it doesn't count (the fix still works).

Reply:
Meh. Still very much not a fan of harcoding key combinations. Can you at least add a small comment there?